spec/models/contributor_spec.rb

```diff
@@ -3,6 +3,22 @@
 require 'rails_helper'
 
 RSpec.describe Contributor, type: :model do
+  shared_examples 'unique within an organization' do
+    before { create(:contributor, organization: organization, **attrs) }
+
+    it 'raises errors when not unique' do
+      expect { create(:contributor, organization: organization, **attrs) }.to raise_error(ActiveRecord::RecordInvalid)
+      expect do
+        build(:contributor, organization: organization, **attrs).save!(validate: false)
+      end.to raise_error(ActiveRecord::RecordNotUnique)
```

Comment:
Why do you feel the need to check the `.save(validate: false)` here?
I assume what is happening here is that by adding validate false, you are bypassing the model unique validations and you are testing that the database index unique validations are preventing a duplicate record from being created even if we were to attempt to do that from the rails console, is that right?

Reply:
Well basically I would have never thought of it, if not finding it on the existing code. What it does though is basically checking if there is a unique index properly set up - as you concluded.